src/test/java/com/atlassian/db/replica/api/AuroraClusterDiscoveryTest.java

```diff
@@ -47,8 +33,7 @@ public void before() throws SQLException {
             "jdbc:postgresql://" + readerEndpoint + "/" + databaseName
         );
         auroraClusterDiscovery = new AuroraClusterDiscovery(
-            anAuroraConnectionDetailsBuilder().username(jdbcUsername).password(jdbcPassword).build()
-        );
+            anAuroraConnectionDetailsBuilder().username(jdbcUsername).password(jdbcPassword).build(), null);
```

Comment:
Why null? Not the test will not test/show new API usage.

Reply:
provided an example usage of the new spi